### PR TITLE
Add Nickname

### PR DESCRIPTION
**Pull request by OzzyCzech**
### Request type

- [x] Feature
- [ ] Fix
- [ ] Refactor
- [x] Tests
- [ ] Documentation

### Summary

According to the [RFC](https://www.rfc-editor.org/rfc/rfc6350#section-6.2.3) there can be `NICKNAME` property

### Change description

Add function `addNickname(nickname: string)` to VCard object

### Check lists

- [ ] Tests passed
- [ ] Coding style respected

### CODE REVIEW

**Comment by OzzyCzech** [comment]
Sorry, my fault, should be separated and fixed now

File: lib/VCard.ts [hunk 1]
```diff
@@ -286,6 +286,16 @@ ${lastName};${firstName};${additional};${prefix};${suffix}\
     return this
   }
 
+  /**
+   * Add nickanem
```
Comment (joaocarmo) [inline lib/VCard.ts:290]: **typo:** `nickanem` instead of `nickname`.

```suggestion
   * Add nickname
```

File: test-functional/test-vcard.js [hunk 2]
```diff
@@ -21,6 +21,7 @@ var suffix = ''
 vCard
   // add personal data
   .addName(lastname, firstname, additional, prefix, suffix)
+  .addNickname('Jero')
```
Comment (joaocarmo) [inline test-functional/test-vcard.js:24]: **suggestion:** can we also add it to the unit tests?

File: lib/VCard.ts [hunk 3]
```diff
@@ -286,6 +286,16 @@ ${lastName};${firstName};${additional};${prefix};${suffix}\
...
+   *
+   * @param {string} nickname
+   */
+  public addNickname(nickname: string): this {
```
Comment (joaocarmo) [inline lib/VCard.ts:294]: **suggestion:** per [RFC 2426, section 3.1.3](https://www.rfc-editor.org/rfc/rfc2426#section-3.1.3), the `nickname` property can support comma-separated values. what do you think of adding support for both `string` and `string[]` types?

**Review by joaocarmo — CHANGES_REQUESTED** [review]
Looking good! Thank you for the quick turnaround.

Left some comments, let me know what you think.

File: lib/VCard.ts (hunk 1 above)
Comment (OzzyCzech) [reply lib/VCard.ts:290]: 🤦

**Review by joaocarmo — APPROVED** [review]
LGTM!

**Pull request merged**